maskit/examples/load_data.py

```diff
@@ -0,0 +1,14 @@
+from maskit.examples.load_circles import load_circles
+from maskit.examples.load_iris import load_iris
+from maskit.examples.load_mnist import load_mnist
+
+
+def load_data(dataset, wires=None, embedding=None, params=None):
```

Comment:
Is the ``embedding`` parameter actually needed anywhere? It seems to be always ignored.

Reply:
It would be great if you could define a ``typing.NamedTuple`` for the return type. You can use that basically for free, but it makes code much easier to understand.

Reply:
@eileen-kuehn @cDenius can we switch the ``params`` to ``**kwargs`` or even explicit keywords here?

This seems to "just" pass through some dict, and from what I can tell it's always the same keywords. I guess taking ``**kwargs`` in the intermediate layers and explicit keywords in the final ``load_XYZ`` functions is a good compromise between simplicity and self-documentation.

Reply:
> @eileen-kuehn @cDenius can we switch the `params` to `**kwargs` or even explicit keywords here?
> 
> This seems to "just" pass through some dict, and from what I can tell it's always the same keywords. I guess taking `**kwargs` in the intermediate layers and explicit keywords in the final `load_XYZ` functions is a good compromise between simplicity and self-documentation.

I think specific keywords are a great idea, also in terms of readability as we then better know which `params` are defined at all.

Reply:
The embedding was intended for some future stuff, but using `**kwargs` it shouldn't matter for now.